$NetBSD: patch-aa,v 1.2 2013/06/11 15:22:47 ryoon Exp $

Find struct timeval by its definition: IRIX's <sys/select.h> names it in select's prototype first.

--- configure.orig
+++ configure
@@ -23557,7 +23557,7 @@
 
 int main(int argc, char **argv) { return 0; }
 _ACEOF
-sim_ac_struct_contents="`$CXXCPP $CPPFLAGS conftest.cpp 2>/dev/null | sed -e '1,/struct[ \t]*timeval/ d' | sed -e '/}/,\$ d'`"
+sim_ac_struct_contents="`$CXXCPP $CPPFLAGS conftest.cpp 2>/dev/null | sed -e '1,/struct[ \t]*timeval[ \t]*{/ d' | sed -e '/}/,\$ d'`"
 rm conftest.cpp
 # canonicalize contents:
 sim_ac_struct_contents="`echo { \$sim_ac_struct_contents }`"
@@ -23597,7 +23597,7 @@
 
 int main(int argc, char **argv) { return 0; }
 _ACEOF
-sim_ac_struct_contents="`$CXXCPP $CPPFLAGS conftest.cpp 2>/dev/null | sed -e '1,/struct[ \t]*timeval/ d' | sed -e '/}/,\$ d'`"
+sim_ac_struct_contents="`$CXXCPP $CPPFLAGS conftest.cpp 2>/dev/null | sed -e '1,/struct[ \t]*timeval[ \t]*{/ d' | sed -e '/}/,\$ d'`"
 rm conftest.cpp
 # canonicalize contents:
 sim_ac_struct_contents="`echo { \$sim_ac_struct_contents }`"
@@ -24066,7 +24066,7 @@
 
 { $as_echo "$as_me:${as_lineno-$LINENO}: checking if user is simian" >&5
 $as_echo_n "checking if user is simian... " >&6; }
-case `hostname -d 2>/dev/null || domainname 2>/dev/null || hostname` in
+case `domainname 2>/dev/null || hostname` in
 *.sim.no | sim.no )
   sim_ac_simian=true
   ;;
